manifests/removed/0000_50_olm_16-packageserver.clusterserviceversion.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: packageserver
+  namespace: openshift-operator-lifecycle-manager
+  annotations:
+    release.openshift.io/delete: "true"
```

Comment:
According to @timflannagan, the package server is still deployed as a csv downstream, so this may conflict. I am spinning up a cluster now.

Reply:
Yeah, the package server component is still deployed using a CSV resource - we had tried to migrate off the CSV and instead deploy it using core k8s manifests, but it extended the scope of the SNO work and there was some complexity involved with migrating existing OLM releases and we didn't want to risk missing the release dates. We'll need to revisit this though, and we should ideally be tracking this $somewhere. cc @perdasilva @kevinrizza

Reply:
The packageServer CSV is actually managed by an OLM controller and is defined [here](https://github.com/openshift/operator-framework-olm/blob/3820d16f4912896c51379297e65563b3ec78d272/pkg/manifests/csv.yaml). CVO cannot be configured to delete this CSV.